### PR TITLE
DX Added

### PR DESCRIPTION
**Pull request by JohnnyMoneJohnny**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/benet.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
         "email": "okbutwhoaskedlikefr@gmail.com"
     },
     "record": {
-        "CNAME": "JohnnyMoneJohnny.github.io"
+        "DX": ["mx1.improvmx.com","mx2.improvmx.com"]
```
Comment (CuteDog5695) [inline domains/benet.json:9]:
```suggestion
        "MX": ["mx1.improvmx.com","mx2.improvmx.com"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/benet.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
         "email": "okbutwhoaskedlikefr@gmail.com"
     },
     "record": {
-        "CNAME": "JohnnyMoneJohnny.github.io"
+        "MX": ["mx1.improvmx.com","mx2.improvmx.com"]
+        "TXT": "v=spf1 include:spf.improvmx.com ~all"
```
Comment (CuteDog5695) [inline domains/benet.json:10]: Just noticed
```suggestion
        "MX": ["mx1.improvmx.com","mx2.improvmx.com"],
        "TXT": "v=spf1 include:spf.improvmx.com ~all"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/benet.json (hunk 2 above)
Comment (JohnnyMoneJohnny) [reply domains/benet.json:10]: I'm dumb af 😭

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**